test/e2e/multicast_test.go

```diff
@@ -56,6 +59,27 @@ func TestMulticast(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Error computing multicast interfaces: %v", err)
 	}
+	t.Run("testMulticastWithNoEncap", func(t *testing.T) {
+		skipIfEncapModeIsNot(t, data, config.TrafficEncapModeNoEncap)
+		runMulticastTestCases(t, data, nodeMulticastInterfaces, true)
+	})
+	t.Run("testMulticastWithEncap", func(t *testing.T) {
+		ac := func(config *agentconfig.AgentConfig) {
+			config.TrafficEncapMode = "encap"
+			config.FeatureGates["Multicast"] = true
```

Comment:
This won't even run since L50 already skips it. BTW, other tests no longer mutate feature gate since https://github.com/antrea-io/antrea/pull/3922

Reply:
Removed.